code/web/src/ui/image/Tile.js

```diff
@@ -4,15 +4,17 @@ import PropTypes from 'prop-types'
 
 // Component
 const Tile = (props) => {
-  const { children, image, width, height, style, shadow, ...others } = props
+  const { children, image, width, height, style, shadow, margin, ...others } = props
 
   return (
-    <div style={Object.assign({ height, width }, style)} {...others}>
+    <div style={Object.assign({ height, width , margin}, style)} {...others}>
       {children}
 
       {/* language=CSS */}
       <style jsx>{`
         div {
+          background-position: center; 
+          background-repeat: no-repeat; 
```

Comment:
While these 2 properties are working, they seem to be causing some extra blank space above and below the picture.

Reply:
I will take a closer look at this today and figure out why.